src/game_inventory.cpp

```diff
@@ -1,21 +1,53 @@
 #include "game.h"
 
+#include "cata_utility.h"
 #include "inventory_ui.h"
 #include "player.h"
 #include "item.h"
 #include "itype.h"
 
+item_location inv_internal( player &p, const inventory_selector_preset &preset,
+                            const std::string &title, int radius,
+                            const std::string &none_message )
+{
+    p.inv.restack( &p );
+    p.inv.sort();
+
+    inventory_pick_selector inv_s( p, preset );
+
+    inv_s.set_title( title );
```

Comment:
> The title might depend upon selector's content (which can be revealed only after construction)

In which case you actually need a `virtual std::string title() const` method.

> This concept is likely to change and more (or less) optional arguments (properties in terms of C#) can persist in the class, but don't worth extracting yet another.

I'm not familiar with C# but 100% of your current instances set title so you can and should do this via the constructor.